README.md

```diff
@@ -83,16 +89,16 @@ Additionally a short, informative sentence must be written as `before_login_hint
 
 ## Configuration Defaults
 
-Beside the server-configuration, Delta Chat has several other options
-that can typically be set by the user at runtime.
+Beside the server-configuration, chatmail/core has several other options
+that can typically be using chatmail apps at runtime.
```

Comment:
I don't understand what this sentence means... aren't configuration parameters set by the user? Are the options using the chatmail apps?

Reply:
this is explained when reading on: you can not only preset server-configuration as` port` or `server` - but also preset eg.` media_quality`, `download_limit`, `delete_server_after`, `mvbox_move`, `sentbox_watch` etc.